src/visualizers/VisualizerGrid.ts

```diff
@@ -0,0 +1,965 @@
+        + ' whether a number is abundant, polygonal, '
+        + 'or in some other sequence.'
+
+    //Grid variables
+    amountOfNumbers = 40000
+    currentSequenceIndex = 0
+    startingIndex = 0
+    currentNumber = 0n
+    showNumbers = false
+    preset = Preset.Custom
+    pathType = PathType.Spiral
+    resetAndAugmentByOne = false
+    squareValues = false
+    backgroundColor = BLACK
+    numberColor = WHITE
+
+    //Path variables
+    x = 0
+    y = 0
+    scalingFactor = 0
+    currentDirection = Direction.Right
+    numberToTurnAtForSpiral = 0
+    incrementForNumberToTurnAt = 1
+    whetherIncrementShouldIncrement = true
+
+    //Properties
+    propertyObjects = [
+        {
+            property: Property.Prime,
+            visualization: PropertyVisualization.Primary_Color,
+            color: RED,
+        },
+    ]
+
+    params: {[key: string]: ParamInterface} = {
+        preset: {
+            value: this.preset,
+            from: Preset,
+            displayName: 'Presets',
+            required: false,
+            description:
+                'Note: If a preset is selected, properties no longer function.',
+        },
+        amountOfNumbers: {
+            value: this.amountOfNumbers,
+            displayName: 'Amount of Numbers',
+            required: false,
+            description:
+                'Note: There is an increased lag time over 10,000 numbers',
+        },
+        startingIndex: {
+            value: this.startingIndex,
+            displayName: 'Starting Index',
+            required: false,
+            description: 'Note: This does not accept negative numbers.',
+        },
+        pathType: {
+            value: this.pathType,
+            from: PathType,
+            displayName: 'How path moves about grid',
+            required: false,
+        },
+        showNumbers: {
+            value: this.showNumbers,
+            forceType: 'boolean',
+            displayName: 'Show numbers',
+            required: false,
+            description:
+                'Note: Show_Path and Show_Numbers default to 400 numbers',
+        },
+        numberColor: {
+            value: this.numberColor,
+            forceType: 'color',
+            displayName: 'Number color:',
+            required: false,
+            visibleDependency: 'showNumbers',
+            visiblePredicate: (dependentValue: boolean) =>
+                dependentValue === true,
+        },
+        backgroundColor: {
+            value: this.backgroundColor,
+            forceType: 'color',
+            displayName: 'Background color:',
+            required: false,
+        },
+    }
+
+    constructor() {
+        super()
+        for (let i = 1; i < MAXIMUM_ALLOWED_PROPERTIES; i++) {
+            this.propertyObjects.push({
+                property: Property.None,
+                visualization: PropertyVisualization.None,
+                color: DEFAULT_COLORS[i],
+            })
+        }
+
```

Comment:
I agree. None is a nonsensical property.

Reply:
Action item: remove the `None` property.

Reply:
OK, I am now working on removing None.

Reply:
Ah, it took me a while to figure out what was meant here. It's not the "color" of the property that's being set to "None", it's the "Visualization" of the property (which otherwise can either be the primary color, which means the whole grid box, or the secondary color, which means a small centered subbox. I will remove the "None" visualization, which is indeed useless.

Reply:
OK, I believe I have resolved this concern. Can someone pull the latest version of this PR and verify things seem to be working as expected without the Visualization option of "None" and then resolve this conversation if so? Thanks.